templates/user/notification/notification_div.tmpl

```diff
@@ -69,50 +69,34 @@
 							</div>
 							<div class="notifications-buttons tw-items-center tw-justify-end tw-gap-1 tw-px-1">
 								{{if ne .Status 3}}
-									<form action="{{AppSubUrl}}/notifications/status" method="post">
-										{{$.CsrfTokenHtml}}
-										<input type="hidden" name="notification_id" value="{{.ID}}">
-										<input type="hidden" name="status" value="pinned">
-										<button class="btn interact-bg tw-p-2" title="{{ctx.Locale.Tr "notification.pin"}}"
-											data-url="{{AppSubUrl}}/notifications/status"
-											data-status="pinned"
-											data-page="{{$.Page.Paginater.Current}}"
-											data-notification-id="{{.ID}}"
-											data-q="{{$.Keyword}}">
-											{{svg "octicon-pin"}}
-										</button>
-									</form>
+									<button type="button" class="btn interact-bg tw-p-2" title="{{ctx.Locale.Tr "notification.pin"}}"
+										hx-post="{{AppSubUrl}}/notifications/status?q={{ $.Keyword }}"
+										hx-target="#notification_div"
+										hx-swap="outerHTML"
+										hx-vals='{"notification_id": "{{.ID}}", "status": "pinned", "page": "{{$.Page.Paginater.Current}}", "_csrf": "{{$.CsrfToken}}", "noredirect": "true" }'
+										{{if eq .Status 3}}disabled{{end}}
+										>
+										{{svg "octicon-pin"}}
+									</button>
```

Comment:
I have some ideas to improve the buttons. Could I push to your branch (if you don't mind)?